src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -1,16 +1,47 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!-- Include common build properties -->
+<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+
   <!-- Compilation options -->
+  <Import Project="../../clr.coreclr.props" />
+  <Import Project="../../clr.defines.targets" />
+
+  <!-- Generate rsp -->
+  <Import Project="GenerateCompilerResponseFile.targets" />
+
+  <Target Name="GenerateTheRSPFile" BeforeTargets="CoreCompile" DependsOnTargets="GenerateCompilerResponseFile">
+  </Target>
+
+  <PropertyGroup>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <GenerateResxSourceOmitGetResourceString>true</GenerateResxSourceOmitGetResourceString>
+    <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
+
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <!-- Default is embedded overwrite this -->
```

Comment:
We should specify that this default is from arcade. As it is now this, comment is incorrect as this change is not yet using arcade.

Reply:
>We should specify that this default is from arcade.

But that is policy and shouldn't be defined in the build environment. Build scripts should define what is desired _not_ the default policy of current build system. I will make this is statement of desire and remove the "default" word.

>As it is now this, comment is incorrect as this change is not yet using arcade.

We actually are using Arcade. All the Arcade targets and settings are being used, even version number generation.